be/src/exprs/vectorized/lambda_function.cpp

```diff
@@ -0,0 +1,50 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+#include "exprs/vectorized/lambda_function.h"
+
+#include <iostream>
+namespace starrocks::vectorized {
```

Comment:
Please format the code with `clang-format`

Reply:
done